server/vcr-server/api/v4/serializers/rest/credential.py

```diff
@@ -45,11 +44,28 @@ class Meta:
             "visible_fields",
             "highlighted_attributes",
             "credential_title",
+            "issuer"
```

Comment:
Should issuer be in the `CredentialTypeExtendedSerializer` ?

Reply:
Yes, it is. Those are the excluded fields for `CredentialTypeSchemaSerializer`, but I'm including it in `CredentialTypeExtendedSerializer` below

Reply:
Ah, right!